components/src/organisms/LabwareDetailsWithCount/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { StyledText } from '../../atoms'
+import { SPACING } from '../../ui-style-constants'
+
+type LabwareDetailsWithCountProps = {
+  title: string
+  subTitle?: string
+  quantity?: string | null
```

Comment:
this type should probably be a number right? also would we ever want to explicitly pass `null` rather than not passing the prop at all?
```suggestion
  quantity?: number
```

Reply:
I wanted it to be a string so we can pass its translation but you are right about the null!

Reply:
i can change the name to tag